src/app/index.ts

```diff
@@ -263,7 +261,8 @@ module.exports = yo.extend({
         let projectRepo = jsonData.getProjectTemplateRepository(this.project.projectType, language == 'ts' ? _.toLower(typescript) : _.toLower(javascript));

```

Comment:
This code could be written more simply. You can have one function that returns the object for a given projectType and language, and that object can have repository and branch properties. If (repository), then use git clone, and then if (branch) then add branch parameter to the git clone command.

Reply:
Fixed as suggested